templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -58,18 +58,28 @@ contents:
     enable_metrics = true
     metrics_port = 9537
     metrics_collectors = [
-      "operations",
-      "operations_latency_microseconds_total",
-      "operations_latency_microseconds",
-      "operations_errors",
       "image_pulls_layer_size",
       "containers_oom_total",
-      "containers_oom",
-      # Drop metrics with excessive label cardinality.
-      # "image_pulls_by_digest",
-      # "image_pulls_by_name",
-      # "image_pulls_by_name_skipped",
-      # "image_pulls_failures",
-      # "image_pulls_successes",
-      # "image_layer_reuse",
+      # "operations", # DEPRECATED
+      # "operations_latency_microseconds_total", # DEPRECATED
+      # "operations_latency_microseconds", # DEPRECATED
+      # "operations_errors", # DEPRECATED
+      # "containers_oom", # DEPRECATED
+      # "image_pulls_by_digest", # DEPRECATED
+      # "image_pulls_by_name",  # DEPRECATED
+      # "image_pulls_by_name_skipped", # DEPRECATED
+      # "image_pulls_failures", # DEPRECATED
+      # "image_pulls_successes", # DEPRECATED
+      # "image_layer_reuse", # DEPRECATED
```

Comment:
Do we still wanna add them for 1-2 releases before disabling completely?

Reply:
Yeah, thanks: I had forgot. We can do that, keep these low-cardinality existing ones enabled here for now (upto next 1-2 releases).
I'll update the PR accordingly.

Reply:
@saschagrunert Made the changes in 4e52dd8 :)